### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by eyass445** [lain-ke/web-os#1]
Did the bot work correctly for you?
https://github.com/chrisleekr/binance-trading-bot

**Comment by lain-ke** [comment]
Yeah I'm currently trying it out but it seems to do the work just fine.

**Comment by eyass445** [comment]
I have a problem ,, not selling upon reaching the sale that I have placed
<img width="394" alt="Screen Shot 2021-03-06 at 1 04 09 PM" src="https://user-images.githubusercontent.com/43729214/110219865-6bae2300-7eca-11eb-997c-7e261d478734.png">
<img width="519" alt="Screen Shot 2021-03-06 at 1 23 10 PM" src="https://user-images.githubusercontent.com/43729214/110219866-6e107d00-7eca-11eb-844d-4b4576aa9164.png">
